indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/Models.kt

```diff
@@ -257,7 +260,7 @@ data class ProofRequestCredentials(
 )
 
 @CordaSerializable
-data class ClaimReferenceInfo(val credInfo: ClaimReference, val interval: String?)
+data class ClaimReferenceInfo(val credInfo: ClaimReference, val interval: Interval? = null)
```

Comment:
When revocation is disabled, there is no need to pass non-revoked interval, because prover won't prove non-revocation of this credential.

AnoncredsDemoTest covers this case.

Reply:
When revocation is disabled, there is no need for a verifier to specify non-revoked interval, because prover won't prove non-revocation anyway.

This case is covered in AnoncredsDemoTest.

Reply:
Could you write in the docs about it, please?